### PR TITLE
rename resources in libnavigation-ui

### PR DESCRIPTION
**Pull request by JunDai**
## Description

A follow-up of https://github.com/mapbox/mapbox-navigation-android/pull/3439 to rename resources in libnavigation-ui

- [x] I have added any issue links
- [x] I have added all related labels (`bug`, `feature`, `new API(s)`, `SEMVER`, etc.)
- [x] I have added the appropriate milestone and project boards

### Implementation

I am referring to [ribot's Android Guidelines](https://github.com/ribot/android-guidelines/blob/master/project_and_code_guidelines.md) as a guideline but made a little change to have `mapbox` as all resources prefix.

For example:
* **layout** - `mapbox_activity_navigation` / `mapbox_notification_navigation_collapsed`
* **icon** - `mapbox_ic_navigation` / `mapbox_ic_cancel`
* **string** - `mapbox_unit_minutes` / `mapbox_end_navigation`
* **style** - `MapboxStyleNotificationTitle` / `MapboxStyleInstructionView`
* ~~**dimen** - `mapbox_image_width_maneuver` / `mapbox_text_height_primary_banner_instruction`~~
* **dimen** - `mapbox_maneuver_image_width` / `mapbox_primary_banner_instruction_text_height`

We can discuss and update the rule if necessary. And later will create a naming convention md file to record this.

## Testing

Please describe the manual tests that you ran to verify your changes

- [x] I have tested locally (including `SNAPSHOT` upstream dependencies if needed) through testapp/demo app and run all activities to avoid regressions
- [x] I have tested via a test drive, or a simulation/mock location app
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have updated the `CHANGELOG` including this PR
- [ ] We might need to update / push `api/current.txt` files after running `$> make core-update-api` (Core) / `$> make ui-update-api` (UI) if there are changes / errors we're 🆗 with (e.g. `AddedMethod` changes are marked as errors but don't break SemVer) 🚀 If there are SemVer breaking changes add the `SEMVER` label. See [Metalava](https://github.com/mapbox/mapbox-navigation-android/blob/master/docs/metalava.md) docs

### CODE REVIEW

File: libnavigation-ui/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -143,24 +143,24 @@
...
         </item>
 
-        <item name="navigationViewRouteStyle">@style/NavigationMapRoute</item>
+        <item name="navigationViewRouteStyle">@style/MapboxStyleNavigationMapRoute</item>
 
-        <item name="navigationViewLocationLayerStyle">@style/NavigationLocationLayerStyle</item>
+        <item name="navigationViewLocationLayerStyle">@style/MapboxStyleNavigationLocationLayerStyle</item>
 
-        <item name="navigationViewDestinationMarker">@drawable/map_marker_dark</item>
+        <item name="navigationViewDestinationMarker">@drawable/mapbox_ic_map_marker_dark</item>
```
Comment (Guardiola31337) [inline libnavigation-ui/src/main/res/values/styles.xml:150]: Could we convert this into a Vector Drawable so we can remove the PNGs? cc @map-angela

File: libnavigation-ui/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -93,18 +93,18 @@
...
         </item>
 
-        <item name="navigationViewRouteStyle">@style/NavigationMapRoute</item>
+        <item name="navigationViewRouteStyle">@style/MapboxStyleNavigationMapRoute</item>
 
-        <item name="navigationViewLocationLayerStyle">@style/NavigationLocationLayerStyle</item>
+        <item name="navigationViewLocationLayerStyle">@style/MapboxStyleNavigationLocationLayerStyle</item>
 
-        <item name="navigationViewDestinationMarker">@drawable/map_marker_light</item>
+        <item name="navigationViewDestinationMarker">@drawable/mapbox_ic_map_marker_light</item>
```
Comment (Guardiola31337) [inline libnavigation-ui/src/main/res/values/styles.xml:100]: Could we convert this into a Vector Drawable so we can remove the PNGs? cc @map-angela

File: libnavigation-ui/src/main/res/values/styles.xml (hunk 1 above)
Comment (JunDai) [reply libnavigation-ui/src/main/res/values/styles.xml:150]: I am going to fix this after https://github.com/mapbox/mapbox-navigation-android/issues/3459 is been addressed.

File: libnavigation-ui/src/main/res/values/styles.xml (hunk 2 above)
Comment (JunDai) [reply libnavigation-ui/src/main/res/values/styles.xml:100]: same as above.

**Comment by JunDai** [comment]
@Guardiola31337 - just pushed two commits:
* to remove the redundant dimens in different screen configuration
* fix #3454 by using the latest icons from UX team.

**Comment by JunDai** [comment]
There are still two **TODOs** which can be done after I get the latest assets from UX team:
* update the map-marker icon by using a vector instead of current png files.
* update the re-center icon since the current one isn't at center.

**Comment by Guardiola31337** [comment]
@JunDai Sounds like a plan 🚀 

> * fix #3454 by using the latest icons from UX team.

Let's make sure we close it after merging here 🎉

**Pull request merged**